op-service/dial/dial.go

```diff
@@ -34,8 +34,8 @@ func DialEthClientWithTimeout(ctx context.Context, timeout time.Duration, log lo
 }
 
 // DialRollupClientWithTimeout attempts to dial the RPC provider using the provided URL.
-// If the dial doesn't complete within timeout seconds, this method will return an error.
-func DialRollupClientWithTimeout(ctx context.Context, timeout time.Duration, log log.Logger, url string) (*sources.RollupClient, error) {
+// If the dial and rpc calls don't complete within timeout seconds, this method will return an error.
+func DialRollupClientWithTimeout(ctx context.Context, timeout time.Duration, log log.Logger, url string, rpcTimeoutCfg ...client.BaseRPCTimeout) (*sources.RollupClient, error) {
```

Comment:
This is a weird place to configure timeouts for `sources.RollupClient`. I see that this is where we are creating it, but if we need to configure it, we should pass a configuration option to it.